src/services/addTokenToExchange.ts

```diff
@@ -0,0 +1,45 @@
+import { ExchangeApi, Erc20Api } from 'types'
+import { getErc20Info } from './getErc20Info'
+import { log, getImageUrl } from 'utils'
+
+interface Params {
+  address: string
+  exchangeApi: ExchangeApi
+  erc20Api: Erc20Api
+}
+
+export async function addTokenToExchange({ address, exchangeApi, erc20Api }: Params): Promise<boolean> {
```

Comment:
Do we plan to use this?

Reply:
Since I was already playing around with the services layer and listing tokens is similar to adding tokens, I implemented it. 
This will not be immediately used. For now is more to get a feeling of the new layer with more than one use case.